### PR TITLE
lsof on umount failure; catpure logs for aggressively

### PR DESCRIPTION
**Pull request by liewegas**
this is trying to track down why we keep seeing that umount hting all the
time!

### CODE REVIEW

**Comment by gregsfortytwo** [comment]
You need to set up PATH or something for the lsof or it will fail on RHEL. :)

**Comment by liewegas** [comment]
fixed!  also added sudo and tested, looks good

File: tasks/ceph.py [hunk 1]
```diff
@@ -76,6 +76,40 @@ def ceph_log(ctx, config):
...
+
+            log.info('Archiving logs...')
+            path = os.path.join(ctx.archive, 'remote')
+            os.makedirs(path)
+            for remote in ctx.cluster.remotes.iterkeys():
+                sub = os.path.join(path, remote.shortname)
+                os.makedirs(sub)
+                teuthology.pull_directory(remote, '/var/log/ceph',
+                                          os.path.join(sub, 'log'))
         pass
```
Comment (andrewschoen) [inline tasks/ceph.py]: Don't need this anymore, right?

File: tasks/ceph.py [hunk 2]
```diff
@@ -728,16 +762,24 @@ def first_in_ceph_log(pattern, excludes):
...
+                        args=[
+                            'sync',
+                            run.Raw('&&'),
+                            'sudo',
+                            'umount',
+                            '-f',
+                            dir_
                         ]
                     )
+                except:
```
Comment (andrewschoen) [inline tasks/ceph.py]: nitpick: bare except

**Pull request merged**